src/main/java/com/mparticle/kits/RadarKit.java

```diff
@@ -40,8 +40,11 @@ private void tryTrackOnce() {
         mRunAutomatically = settings.containsKey(KEY_RUN_AUTOMATICALLY) && Boolean.parseBoolean(settings.get(KEY_RUN_AUTOMATICALLY));
 
         Radar.initialize(context, publishableKey);
+        
+        // TODO proactively get customer ID and call Radar.setUserId(customerId)?
```

Comment:
@nickpatrick if you are looking to see if the CustomerId is present when the kit is created, you can call:

```
Map<MParticle.IdentityType, String> identities = getUserIdentities();
        if (identities.containsKey(MParticle.IdentityType.CustomerId)) {
            ...do something
        }
```